src/Command/ImportTimesheetCommand.php

```diff
@@ -318,11 +318,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     foreach (explode(',', $record['Tags']) as $tagName) {
                         if (empty($tagName)) {
                             continue;
-                        } elseif (!empty($tag = $this->tagRepository->findTagByName($tagName))) {
-                            $timesheet->addTag($tag);
-                        } else {
-                            $timesheet->addTag((new Tag())->setName($tagName));
+                        } 
+                        
+                        if (null === ($tag = $this->tagRepository->findTagByName($tagName)) {
```

Comment:
A closing bracket is missing.

Reply:
Seems i can't further commit here or do i miss a something?

Reply:
Yeah, the suggestion feature here in the comments is tricky, thanks for fixing it manually!